test/main.py

```diff
@@ -22,86 +23,61 @@ def generate_indent(number):
 
 class CiscoConfig:
     nodes = []
+    interfaces = []
+    root = None
+    indent = None
+    parent = None
+    config = None
+    i = None
 
     def __init__(self, file):
         self.file = file
         self.fd = open(file, "r")
-        root = Node("root")
-        config = self.create_entry_list(self.fd.readlines())
-
-        i = iter(config)
-        indent = 0
-        parent = root
+        self.config = self.create_entry_list(self.fd.readlines())
```

Comment:
Tą funkcję bym wywalił bo nie potrzebujemy tu preprocesingu. Wrzuć ta funkcjonalność w metodę parse. Przy przypisaniu iteratora do line usuń \n. Następnie następnie jeżeli line zaczyna się od znaku komentarza to daj continue. Zapisz znak komentarz w zmiennej w ciele klasy. Np. COMMENT_SYMBOL = "!". Ułatwia to czytelność kodu.